packages/appeals-service-api/src/routes/v2/users/index.js

```diff
@@ -1,12 +1,16 @@
 const express = require('express');
-const { userGet, userPost, userLink } = require('./controller');
+const { userPost, userSearch, userGet, userUpdate, userDelete, userLink } = require('./controller');
 const router = express.Router();
 const asyncHandler = require('@pins/common/src/middleware/async-handler');
+const { openApiValidatorMiddleware } = require('../../../validators/validate-open-api');
 
-// Would be nice to untangle this at some point, we're using
```

Comment:
do we want to allow either like I have done here

Reply:
Works for me!